### PR TITLE
Configure a copy of the config service to run on Fargate

### PR DESCRIPTION
**Pull request by Krenair**
You may want to use `diff -rb terraform/modules/hub/modules/ecs_app terraform/modules/hub/modules/ecs_fargate_app` to review part of this PR. By and large it's the existing thing with some tweaks to send it to Fargate instead of EC2.

### CODE REVIEW

File: terraform/modules/hub/modules/ecs_fargate_app/variables.tf [hunk 1]
```diff
@@ -0,0 +1,66 @@
+variable "cluster" {}
```
Comment (blairboy362) [inline terraform/modules/hub/modules/ecs_fargate_app/variables.tf:1]: Maybe rename this to `service`

File: terraform/modules/hub/modules/ecs_fargate_app/ecs.tf [hunk 2]
```diff
@@ -0,0 +1,41 @@
+module "cluster_ecs_roles" {
+  source = "../ecs_iam_role_pair"
+
+  deployment       = var.deployment
+  service_name     = var.cluster
+  tools_account_id = var.tools_account_id
+  image_name       = var.image_name
+}
+
+resource "aws_ecs_task_definition" "cluster" {
```
Comment (blairboy362) [inline terraform/modules/hub/modules/ecs_fargate_app/ecs.tf:10]: `cluster` feels wrong for the name here. Maybe 'service' or `app`? Not sure...

File: terraform/modules/hub/modules/ecs_fargate_app/ecs.tf [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = var.cpu
+  memory                   = var.memory
+}
+
+output "task_role_name" {
+  value = module.cluster_ecs_roles.task_role_name
+}
+
+resource "aws_ecs_service" "cluster" {
```
Comment (blairboy362) [inline terraform/modules/hub/modules/ecs_fargate_app/ecs.tf:25]: `service` or `app` or something?

File: terraform/modules/hub/modules/ecs_fargate_app/lb.tf [hunk 4]
```diff
@@ -0,0 +1,63 @@
+resource "aws_lb" "cluster" {
```
Comment (blairboy362) [inline terraform/modules/hub/modules/ecs_fargate_app/lb.tf:1]: `service` or `app`?

File: terraform/modules/hub/modules/ecs_fargate_app/route53.tf [hunk 5]
```diff
@@ -0,0 +1,23 @@
+resource "aws_route53_zone" "cluster" {
```
Comment (blairboy362) [inline terraform/modules/hub/modules/ecs_fargate_app/route53.tf:1]: Is this needed?

File: terraform/modules/hub/modules/ecs_fargate_app/security_groups.tf [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+  description = "${local.identifier}-lb"
+
+  vpc_id = var.vpc_id
+}
+
+output "lb_sg_id" {
+  value = aws_security_group.lb.id
+}
+
+resource "aws_security_group_rule" "task_ingress_from_lb" {
```
Comment (blairboy362) [inline terraform/modules/hub/modules/ecs_fargate_app/security_groups.tf:12]: can `microservice_connection` be used in place of these two security group rules?

File: terraform/modules/hub/modules/ecs_fargate_app/route53.tf (hunk 5 above)
Comment (Krenair) [reply terraform/modules/hub/modules/ecs_fargate_app/route53.tf:1]: Maybe. Everything in this module is a copy of the equivalent from ecs_app, and unless there's a Fargate-specific reason to change it I'd prefer to keep it the same for now.

**Pull request merged**